e2e-vm/setup-vm.sh

```
@@ -0,0 +1,55 @@
+    --ssh-inject testuser:file:"$KEY.pub" \
+    --run-command 'apt-get update' \
+    --install 'gnome-shell,gnome-shell-common,gnome-session,glib2.0-bin,dbus,mesa-utils,libgl1-mesa-dri,libgbm1,dconf-gsettings-backend,gsettings-desktop-schemas,libportaudio2,curl,pulseaudio,pulseaudio-utils' \
+    --run-command 'systemctl set-default graphical.target' \
+    --run-command 'mkdir -p /etc/gdm3' \
+    --upload /dev/stdin:/etc/gdm3/custom.conf <<'GDMEOF'
+[daemon]
+AutomaticLoginEnable=True
+AutomaticLogin=testuser
+WaylandEnable=true
+GDMEOF
+    --run-command 'rm -f /etc/ssh/ssh_host_*; ssh-keygen -A' \
```

Shellcheck (0.11.0)

[warning] This flag is used as a command name. Bad line break or missing [ .. ]?

(SC2215)